src/assets/stylesheets/custom.scss

```diff
@@ -68,6 +68,9 @@ $textCenter: center;
 .dib{ display: inline-block; }
 .db{ display: block; }
 .textCenter{ text-align: $textCenter; }
+.pointer{ cursor: pointer; }
+.weight-600{ font-weight: 600; }
```

Comment:
We need these properties at many places that why I added it here in custom.scss file.